app/build.gradle

```diff
@@ -34,21 +35,15 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
-
-    buildFeatures {
-        viewBinding true
-    }
-   namespace 'com.example.life_is_well'
```

Comment:
We wanna make sure this is still in the project.

Reply:
Yes, I was able to double check this.